test.js

```diff
@@ -1,10 +1,102 @@
 import test from 'ava';
-import fn from './';
+import serialize from './';
 
 test(t => {
-	const x = Object.keys(fn(new Error('foo')));
-	t.true(x.includes('name'));
-	t.true(x.includes('stack'));
-	t.true(x.includes('message'));
+	const serialized = serialize(new Error('foo'));
+	const x = Object.keys(serialized);
+	t.not(x.indexOf('name'), -1);
```

Comment:
Looks like I found an AVA regression here.

`x.includes` stopped working for me